### PR TITLE
add test for JointTrajectoryClientsContainer

### PR DESCRIPTION
**Pull request by chatblanc-ciel**

### CODE REVIEW

File: arci/src/joint_trajectory_clients_container.rs [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+mod tests {
+    #[test]
+    fn test_container_new() {
+        use super::*;
+        #[derive(Debug, Clone)]
+        struct Dammy {
+            name: Vec<String>,
+        }
+        #[async_trait]
+        impl JointTrajectoryClient for Dammy {
```
Comment (OTL) [inline arci/src/joint_trajectory_clients_container.rs:120]: Dammy -> Dummy

**Comment by OTL** [comment]
arci/src/joint_trajectory_clients_container.rs -> arci/src/clients/joint_trajectory_clients_container.rs

File: arci/src/clients/joint_trajectory_clients_container.rs [hunk 2]
```diff
@@ -1,7 +1,6 @@
 use crate::error::Error;
 use crate::traits::{JointTrajectoryClient, TrajectoryPoint};
-use crate::waits::WaitFuture;
-use futures::stream::FuturesOrdered;
+use async_trait::async_trait;
 
 pub struct JointTrajectoryClientsContainer<T: JointTrajectoryClient> {
```
Comment (OTL) [inline arci/src/clients/joint_trajectory_clients_container.rs:5]: Please do not change except for your test.

Comment (chatblanc-ciel) [reply arci/src/clients/joint_trajectory_clients_container.rs]: I fixed it.
Sorry data was disturbed, these codes are old one.

**Review by OTL — APPROVED** [review]
👍🏼👍🏼👍🏼

**Pull request merged**